### PR TITLE
ID:FPCO-25524;DONE:90;HOURS:1; Remove set env by name option

### PR DESCRIPTION
**Pull request by ManishPrajapati-GoFynd**

### CODE REVIEW

File: src/fdk.ts [hunk 1]
```diff
@@ -211,8 +211,17 @@ export async function init(programName: string) {
...
-    if (!configStore.get(CONFIG_KEYS.CURRENT_ENV_VALUE))
-        configStore.set(CONFIG_KEYS.CURRENT_ENV_VALUE, 'fynd');
+    const current_env = configStore.get(CONFIG_KEYS.CURRENT_ENV_VALUE)
+
+    if (!current_env || !current_env.includes("api."))
+    configStore.set(CONFIG_KEYS.CURRENT_ENV_VALUE, 'api.fynd.com');
+
+    // todo: remove this warning in future version of fdk cli, whem everybody get used to set env by url.
+    if(current_env && !current_env.includes("api.")){
+        console.warn(chalk.yellow('Warning: Reseting active environment to api.fynd.com. Please use `fdk env set -u <env-api-url>` to change active environment.'))
```
Comment (jinalvirani) [inline src/fdk.ts]: Add documentation link

File: tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
     // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
-    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
+    "sourceMap": true,                     /* Generates corresponding '.map' file. */
```
Comment (jinalvirani) [inline tsconfig.json]: Please revert this

File: src/commands/environment/environment-builder.ts [hunk 3]
```diff
@@ -8,11 +10,15 @@ export default function environmentCommandBuilder() {
...
     env.command('ls')
         .description('List supported environments')
-        .asyncAction(Env.listEnvs);
+        .action(()=>{
+            console.warn(chalk.yellow(`Warning: The "env ls" command is deprecated. Ref: ${getPlatformUrls().partners}/help/docs/partners/themes/vuejs/command-reference#environment-commands-1`));
+        });
 
     env.command('set')
+        // todo: remove -n option in 4.0.6
         .option('-n, --name [env-name]', 'Environment name')
```
Comment (vivek-gofynd) [inline src/commands/environment/environment-builder.ts:22]: @manishprajapati52 @jinalvirani aren't we removing env set option here ?

Comment (jinalvirani) [reply src/commands/environment/environment-builder.ts:22]: To show deprecation warning we have kept the option as it is. If you check the logic, it has condition to show env set -n command is deprecated.

**Review by vivek-gofynd — APPROVED** [review]

**Review by vivek-gofynd — APPROVED** [review]

**Pull request merged**